neo4j/neo4j-browser
# Switch back to using injected Desktop API

Due to issues with Desktop's implementation to follow the schema.

It's a large PR, but most of it comes back from being deleted in https://github.com/neo4j/neo4j-browser/pull/979. Some parts have been refactored to make it cleaner and more separated into it's own module.

Now most of the code is in `modules/desktop-api`, except implementations on what do when certain events happen.

### How to test
To test this out, run the dev server `yarn start` and in Neo4j Desktop add the url to the dev server as a graph-app and see it connect to the current running graph when you open the app.
Or build and pack it and install it as a gzip packaged in Neo4j Desktop graph app and try it out.
